CodeMaid.config

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I guess you are using CodeMaid. This file probably doesn't need to be here. If it keeps getting added, feel free to add it to the .gitignores.